app/helpers/importmap/importmap_tags_helper.rb

```diff
@@ -10,7 +18,7 @@ def javascript_importmap_tags(entry_point = "application", importmap: Rails.appl
 
   # Generate an inline importmap tag using the passed `importmap_json` JSON string.
   # By default, `Rails.application.importmap.to_json(resolver: self)` is used.
-  def javascript_inline_importmap_tag(importmap_json = Rails.application.importmap.to_json(resolver: self))
+  def javascript_inline_importmap_tag(importmap_json)
```

Comment:
Can you explain why you removed this default value declaration?

Reply:
Sure, I removed it since `Rails.application.importmap` doesn't exist anymore.
I would even go one step further and pass in an `importmap` instead of `importmap_json` as the parameter, similar to what `javascript_importmap_module_preload_tags` accepts, which would be cleaner IMO.
What do you think?

Reply:
Yes. Or maybe something like:
```suggestion
  def javascript_inline_importmap_tag(importmap_json)
    importmap_json ||= Rails.application.importmaps.find do |importmap|
      importmap.name == "application"
    end.to_json(resolver: self)
```
Assuming `importmap` has an attribute `name` or something like that for identification.

Reply:
I went with passing the importmap as a param, since `javascript_importmap_module_preload_tags` accepts that as well. It looks very clean IMO, what do you think?

Reply:
I don't have any particular preference. Let's see what others think.